ospo-book/chapters/02-chapter.md

```diff
@@ -1 +1,107 @@
-TBD
\ No newline at end of file
+# Chapter 2: Understanding the Value of Open Source Program Offices
+
+- [Introduction](#introduction)
+    - Reasons Behind Starting an OSPO
+    - Reasons Behind Sustaining an OSPO
+- [Assessing Value of Open Source Activity](#assessing-value-of-open-source-activity) - `✅ Assessment`
+- [Antipatterns](#antipatterns) - `🚫 OSPO Antipatterns`
+- [Resources](#resources) - `📚 Continue Here`
+
+# Introduction
+
+From end-user companies and software companies to universities and public administrations, an organization's relationship with open source and the value it can get from an OSPO depend on the type of organization it is. However, we can still identify some critical problems that arise when getting involved in open source on an organizational level, which motivates the creation and continuity of an OSPO.
+
+* **The tension between the need to ship product features and the need to contribute back to open source:** Open source contributions may take a back seat when dealing with multiple day-to-day tasks.
+
+* **Procurement processes with never-ending steps:** Open source is a dynamic ecosystem and contributions to it should occur as smoothly and naturally as possible. The long procurement processes faced in highly regulated environments, such as finance companies and governments, create a barrier to open source contribution and engagement.
+
+* **A lack of consciousness about organizational responsibility:** Due to the way open source was taught in the past, engineering-based tools, or even the engineering jargon used, the concept of open source may not be taken seriously in other areas of the organization involved in decision-making processes, management, or policy making.
+
+
+To fully overcome these and other problems when engaging in open source, organizations need to be equipped to manage open source operations effectively on both cultural and practical levels.
+
+To understand the value of an OSPO, the reader needs to get familiar to the reasons behind (1) starting an OSPO and (2) sustaining an OSPO in the long term
```

Comment:
I'm not sure the reasons to start an OSPO are much different from sustaining an OSPO. I would more look at it as reasons for **having** an OSPO.

Side note: The "program" in OSPO might mislead some organizations into thinking that this is a temporary activity which has an end, when the program is finished. That's why I would focus on the responsibility of an OSPO to manage open source as the ongoing activity which might evolve over time to become more sophisticated, but is certainly not a time-limited activity with a projected end.